<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>261</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>choice1</string>
         </property>
         <widget class="QRadioButton" name="startframe">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>60</y>
            <width>147</width>
            <height>15</height>
           </rect>
          </property>
          <property name="text">
           <string>startframe</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="endframe">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>160</y>
            <width>147</width>
            <height>15</height>
           </rect>
          </property>
          <property name="text">
           <string>endframe</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>choice2</string>
         </property>
         <widget class="QRadioButton" name="inner">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>60</y>
            <width>147</width>
            <height>15</height>
           </rect>
          </property>
          <property name="text">
           <string>inner</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="vector">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>150</y>
            <width>147</width>
            <height>15</height>
           </rect>
          </property>
          <property name="text">
           <string>vector</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>471</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>773</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
